piker/data/feed.py

```diff
@@ -309,6 +281,11 @@ async def start_backfill(
             - pendulum.from_timestamp(times[-2])
         ).seconds
 
+        if step_size_s == 60:
+            inow = round(time.time())
+            if (inow - times[-1]) > 60:
+                await tractor.breakpoint()
```

Comment:
Should we keep this `breakpoint`?

Reply:
good catch. no should probably not since i don't think this issue, when it shows up is a major problem.